src/main/java/cascading/tap/hive/HiveConfUtil.java

```diff
@@ -0,0 +1,16 @@
+package cascading.tap.hive;
```

Comment:
I am going to remove it from the commit. I will have to reformat it anyway. The attribution will stay intact.